### PR TITLE
[F] Localize backend components

### PR DESCRIPTION
**Pull request by LeonorPDX**

### CODE REVIEW

File: client/src/backend/components/analytics/parts/chartOptions.js [hunk 1]
```diff
@@ -44,7 +54,7 @@ const rechartsOptions = ({ tooltipLabel }) => ({
     type: "number",
     scale: "time",
     stroke: AXIS_COLOR,
-    tickFormatter: value => formatDate(value, "MM/dd"),
+    tickFormatter: value => formatDate(value, "MM/dd"), // TODO: Localize date ticks, refactor required
```
Comment (dananjohnson) [inline client/src/backend/components/analytics/parts/chartOptions.js:57]: @1aurend from the l10n work you did with other dates in the app, can you think of an easy way to localize these x-axis labels?
Screen Shot 2022-04-25 at 11 09 31 AM

File: client/src/backend/components/category/List/TextInner.js [hunk 2]
```diff
@@ -94,7 +96,7 @@ export default class TextInner extends Component {
               )}
             </h3>
             <span className="texts-list__date">
-              <FormattedDate
+              <FormattedDate // Ask Lauren: Is this all set to translate? Any changes I need to make?
```
Comment (dananjohnson) [inline client/src/backend/components/category/List/TextInner.js:99]: @1aurend flagging this comment for you.

**Review by dananjohnson — CHANGES_REQUESTED** [review]
I mainly clicked around to check for any issues in the client. Just caught one! Nice work on this huge PR, @LeonorPDX! 👏🏻 

Screen Shot 2022-04-25 at 11 29 37 AM

File: client/src/backend/components/analytics/parts/chartOptions.js (hunk 1 above)
Comment (1aurend) [reply client/src/backend/components/analytics/parts/chartOptions.js]: Just pushed a solution for this and couple other little l10n things in the chart.

**Review by dananjohnson — APPROVED** [review]

**Pull request merged**